src/components/Table/Table/__stories__/table.stories.mdx

```diff
@@ -0,0 +1,187 @@
+          dataState={{ isError: false, hasMoreResults: false }}
+          errorState={<TableErrorState />}
+          emptyState={<TableEmptyState />}
+          columns={emailColumns}
+        >
+          <TableHeader>
+            {emailColumns.map((headerCell, index) => (
+              <TableHeaderCell
+                key={index}
+                title={headerCell.title}
+                onSortClicked={sortState => onSort(headerCell.id, sortState)}
+                sortState={sorting[headerCell.id]}
+              ></TableHeaderCell>
+            ))}
+          </TableHeader>
+          <TableBody>
+            {tableData.map(rowItem => (
+              <TableRow key={rowItem.id}>
+                <TableCell>{rowItem.sentOn}</TableCell>
+                <TableCell>{rowItem.subject}</TableCell>
+                <TableCell>
+                  <TableAvatar text={rowItem.sentBy} />
+                </TableCell>
+                <TableCell>
+                  <Label text={rowItem.status} isAnimationDisabled />
+                </TableCell>
+                <TableCell>{rowItem.emailsSent}</TableCell>
+              </TableRow>
+            ))}
+          </TableBody>
+        </Table>
+      );
+    }};
+  </Story>
+</Canvas>
+
+## Props
+<ArgsTable of={ Table } />
+
+## Loading
+Using skeleton
+<Canvas>
+  <Story name="Loading">
+    {() => {
+      return (
+        <Table
+          dataState={{ isError: false, hasMoreResults: false }}
+          errorState={<TableErrorState />}
+          emptyState={<TableEmptyState />}
+          columns={emailColumns}
+        >
+          <TableHeader>
+            {emailColumns.map((headerCell, index) => (
+              <TableHeaderCell
+                key={index}
+                title={headerCell.title}
+              ></TableHeaderCell>
+            ))}
+          </TableHeader>
+          <TableBody>
+            {[...new Array(5)].map((key, index) => (
+              <TableRow key={key}>
+                <TableCell>
+                  <Skeleton type={Skeleton.types.TEXT} height={20} width={index % 2 === 0 ? 140 : 70} />
+                </TableCell>
+                <TableCell>
+                  <Skeleton type={Skeleton.types.TEXT} height={20} width={index % 2 === 0 ? 200 : 120} />
+                </TableCell>
+                <TableCell>
+                  <Skeleton type={Skeleton.types.CIRCLE} height={24} width={24} />
+                </TableCell>
+                <TableCell>
+                  <Skeleton type={Skeleton.types.TEXT} height={20} />
+                </TableCell>
+                <TableCell>
+                  <Skeleton type={Skeleton.types.TEXT} height={20} />
+                </TableCell>
+              </TableRow>
+            ))}
+          </TableBody>
+        </Table>
+      );
+    }};
+  </Story>
+</Canvas>
+
+## Empty State
+Table has no data to show
+<Canvas>
+  <Story name="Empty State">
+    <Table {...basicTableProps} emptyState={<TableEmptyState />}>
```

Comment:
basicTableProps will not be accessible to the documentation readers and they will be able to recreate this story by copy paste. Let's show all the required props for recreate this story.